### PR TITLE
Upgrade material-ui to @material-ui/core

### PR DESCRIPTION
**Pull request by fragm3**
Fixes #2079

Changes: 

- Upgrade Material UI
- Use MaterialMUITheme for theming #2082, will extend it in upcoming PR's
 - Create Common UI Components #2081, using styled-components, extend it further in next PR's
- Use `rem` unit instead of pixel ##2088, extends in next PR's

Demo Link: https://pr-2079-2-fossasia-susi-web-chat.surge.sh

Components To be Done in next PR:

 - Settings Left Menu
 - StaticAppBar
 - Blogs

Note: I will be improving UI/UX in subsequent PR's. Material Theming will be extended more

### CODE REVIEW

**Comment by fragm3** [comment]
@akshatnitd @harsh-jindal please review

**Comment by fragm3** [comment]
@akshatnitd resolved the conflicts

**Comment by fragm3** [comment]
@akshatnitd I will be extending common UI components in upcoming PR's extensively

**Comment by fragm3** [comment]
@akshatnitd It is working, see again please.

File: src/__tests__/components/TableComplex/RemoveDeviceDialog.js [hunk 1]
```diff
@@ -1,9 +0,0 @@
-import React from 'react';
```
Comment (akshatnitd) [inline src/__tests__/components/TableComplex/RemoveDeviceDialog.js]: Why this is removed?

**Comment by fragm3** [comment]
@akshatnitd added the test file

**Comment by fragm3** [comment]
@akshatnitd will keep that in mind.

**Pull request merged**